src/controller/scan/base_controller.go

```diff
@@ -815,18 +815,26 @@ func (bc *basicController) GetVulnerable(ctx context.Context, artifact *ar.Artif
 
 		var severity vuln.Severity
 
-		for _, v := range vuls {
-			if allowlist.Contains(v.ID) {
-				// Append the by passed CVEs specified in the allowlist
-				vulnerable.CVEBypassed = append(vulnerable.CVEBypassed, v.ID)
+		if allowlistIsExpired {
+			for _, v := range vuls {
+				if severity == "" || v.Severity.Code() > severity.Code() {
+					severity = v.Severity
+				}
+			}
+		} else {
+			for _, v := range vuls {
+				if allowlist.Contains(v.ID) {
```

Comment:
If change this line to `if !allowlistIsExpired && allowlist.Contains(v.ID) {}`, then line 818-823 can be removed as logic duplicated with line 835-837?

Reply:
@hyeyoung-leee could you please update the code to resolve this comment? thanks

Reply:
@chlins @zyyw 
Thanks for the comment.
I updated the code based on the comment.